src/libstd/path.rs

```diff
@@ -1419,20 +1423,28 @@ impl<'a, T: ?Sized + AsRef<OsStr>> From<&'a T> for PathBuf {
 
 #[stable(feature = "rust1", since = "1.0.0")]
 impl From<OsString> for PathBuf {
+    /// Converts a `OsString` into a `PathBuf`.
```

Comment:
I'm not sure if we need this sentence, but let's hear opinions. It's basically restating the code already says.

Reply:
The format is generally "summary sentence, then other stuff". If you want other stuff, you need a summary. I'm not sure how else you'd summarize this. TL;DR it's fine.